### PR TITLE
fix: bottom tooltip position

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Small fix for the bottom position feed alert
Screenshot 2022-11-17 at 13 43 02

Screenshot 2022-11-17 at 13 43 09

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-737 #done

### CODE REVIEW

File: packages/shared/src/components/filters/MyFeedHeading.tsx [hunk 1]
```diff
@@ -39,7 +39,12 @@ function MyFeedHeading({
       offset={[4, 8]}
       isAlertDisabled={isAlertDisabled}
       onClose={() => onUpdateAlerts({ myFeed: null })}
-      className={{ label: 'w-44', message: 'ml-4', wrapper: 'mr-auto' }}
+      className={{
+        label: 'w-44',
+        message: !sidebarRendered ? 'ml-4' : null,
+        wrapper: 'mr-auto',
+        container: !sidebarRendered ? 'left-0' : null,
+      }}
```
Comment (sshanzel) [inline packages/shared/src/components/filters/MyFeedHeading.tsx:47]: I believe you can just use the `offset` property to make adjustments to the whole pointer.
`offset={[8, 8]}`

Though I think the code in the `AlertPointer` is missing this.
In the attached image, I have modified the `bottom` property to include the unused offset parameter. The other ones should also apply the unused parameters.
image

Either way, the proposed solution here fixes the concern, so I'll leave it up to you if you would want to update the `AlertPointer` file.

**Review by sshanzel — APPROVED** [review]
Looks good to me! Just had one comment around the `AlertPointer`.

File: packages/shared/src/components/filters/MyFeedHeading.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/filters/MyFeedHeading.tsx:47]: Ah good point, let me modify that.

**Review by sshanzel — APPROVED** [review]
Thank you so much for fixing the AlertPointer itself 🙇 🙏

**Pull request merged**